api/handlers.go

```diff
@@ -86,6 +100,27 @@ func (sl *skylink) UnmarshalJSON(b []byte) error {
 	return nil
 }
 
+// blocklistGET returns a list of blocked hashes and associated tags
+func (api *API) blocklistGET(w http.ResponseWriter, r *http.Request, _ httprouter.Params) {
+	blocked, err := api.staticDB.BlockedHashes()
+	if err != nil {
+		skyapi.WriteError(w, skyapi.Error{err.Error()}, http.StatusInternalServerError)
+		return
+	}
+
+	hashes := make([]BlockedHash, len(blocked))
+	for i, bh := range blocked {
+		hashes[i] = BlockedHash{
+			Hash: bh.Hash.Hash,
+			Tags: bh.Tags,
+		}
+	}
+	skyapi.WriteJSON(w, BlocklistGET{
+		Hashes:  hashes,
+		Created: time.Now().UTC(),
+	})
```

Comment:
This response is going to return a significant amount of data. We should offer pagination on this endpoint. This will alleviate the load on both our end (server and database) and the client's end.

Reply:
I don't mind paging, we just usually don't do paging, here it makes sense though. I'm more inclined to take into account a date offset, the caller could use the timestamp it received in an initial request and use it as an offset in consecutive requests.

Reply:
A date offset is very nice for subsequent syncs. For the first one it doesn't help much.

Reply:
I'll just do both

Reply:
I'd just start with `offset` + `limit` for now. No need to do timestamps atm if we are only going to use one anyway.